### ISSUE TITLE
Review All Table Column Names

### ISSUE THREAD

**Issue by djbears32** [VMormoris/tw2-stats#3]
1. Capitalize all Column Headers
2. Possible Rename?

**Comment by VMormoris** [comment]
1. If you mean the first letter of each word. I would say that is probably a great idea. But I don't think capitalizing the whole word(s) of its column header would look that good.
2. After reading your issue, I revisit the tables and watch more carefully at the column header(s). And I notice that some columns are named diffidently across tables. Having that in mind I believe that renaming some of the columns is essential for consistency's sake. Would you be kind enough to compile a list with the column header(s) that you want see renamed so we can discuss this further before we make any changes?

**Comment by djbears32** [comment]
yea when i get to this ill put a list in here of what is it and what i suggest naming it to

**Comment by ManourasM** [comment]
I agree with Capitalizing at least the first letter in Column headers. Also some small mistakes to Header Names I found with a quick look:
1. Spelling mistake in tribes table: **victory** points
2. Inconsistent spaces on **off bash** and **def bash** Columns between tables

**Comment by VMormoris** [comment]
It took me a long time, but with the recent pull request #7 I manage to handle your request.
* Capitalized the first letter in every table
* Changes the names so they are consistent across different tables.
* Rename offbash, defbash and totalbash to OBP, DBP and TBP respectively.

_You can check out those and all the other changes I made at: [dev.tw2-stats.com](https://dev.tw2-stats.com)_
If you think that I should rename something else please don't hesitate to tell me.

**Comment by djbears32** [comment]
Nice I'll take a look in a bit here and try and be a bit more focused on this stuff.
